tempest/api/workloadmgr/workload_reassign/test_workload_reassign.py

```diff
@@ -72,9 +114,178 @@ def test_workload_reassign(self):
                 LOG.error("Workload reassign from tenant 2 to 1 failed")
                 reporting.add_test_step(
                     "Workload reassign from tenant 2 to 1", tvaultconf.FAIL)
+
+            #check if actually tenant id is changed for the workload or not. 
+            workload_instance_info = self.get_workload_details(workload_id)
+            LOG.debug(f"workload_instance teanant id={workload_instance_info['project_id']} and tempest conf tenant id={tenant_id}")
+            if tenant_id == workload_instance_info['project_id']:
+                LOG.debug("Workload instance having correct tenant id. TEST CASE PASSED")
+                reporting.add_test_step(
+                    "tenant_id workload reassign", tvaultconf.PASS)
+            else:
+                LOG.error("Workload instance showing different tenant id. TEST CASE FAILED")
+                raise Exception("tenant_id workload reassign")
+
+
+        except Exception as e:
+            LOG.error("Exception: " + str(e))
+            reporting.set_test_script_status(tvaultconf.FAIL)
             reporting.test_case_to_write()
 
+        finally:
+            reporting.test_case_to_write()
+
+    '''
+    OS-2058 - workload reassign with same user and same project.
+    http://192.168.15.51/testlink/linkto.php?tprojectPrefix=OS&item=testcase&id=OS-2058
+    '''
+    @decorators.attr(type='workloadmgr_api')
+    def test_02_workload_reassign(self):
+        try:
+
+            reporting.add_test_script(str(__name__) + "_same_user_same_tenant")
+
+            ### Create vm and workload ###
+            LOG.debug("Create VM")
+
+            self.created = False
+            vm_id = self.create_vm(vm_cleanup=True)
+            LOG.debug("\nVm id : {}\n".format(str(vm_id)))
+
+            LOG.debug("Create workload")
```

Comment:
Remove any unwanted logging statements, which would make it difficult to trace the tempest log for any errors.